Alpaca-Finance-Agents/AllocPoint-Change-Alert/src/agent.ts

```diff
@@ -0,0 +1,102 @@
+import { 
+  Finding, 
+  HandleTransaction, 
+  TransactionEvent, 
+  FindingSeverity, 
+  FindingType,
+  LogDescription
+} from 'forta-agent';
+import { 
+  decodeParameters
+} from "forta-agent-tools";
+
+const POOL_OWNERS: string[] = [
+  "0xc48FE252Aa631017dF253578B1405ea399728A50", // MDEX - BscPool
+  "0xA1f482Dc58145Ba2210bC21878Ca34000E2e8fE4"  // PancakeSwap - Timelock
+]
+
+export const setFuncAbi: string = "function set(uint256 _pid, uint256 _allocPoint, bool _withUpdate)";
+export const setFuncSig: string = "set(uint256,uint256,bool)";
+
+const queueTxnAbi: string = "event QueueTransaction(bytes32 indexed txHash, address indexed target, uint value, string signature, bytes data, uint eta)";
+
+
+function isAddressRelevant(contractAddress: string | null, poolArray: string[]): boolean {
+  if(contractAddress && poolArray.includes(contractAddress)) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
  return contractAddress && poolArray.includes(contractAddress) 
  }
```
When you see the patter `if (<boolean_expresion>) return true; else return false` that can be usually changed by `return <boolean_expresion>`